index.js

```diff
@@ -1,5 +1,5 @@
 let { readFileSync } = require("fs");
-let regex = /sourceMappingURL=data:application\/json;base64,(\S+)/;
+let regex = /sourceMappingURL=data:application\/json;(?:charset=utf-8;)base64,(\S+)/;
```

Comment:
I assume node-sass does not include the charset in the media type, so maybe the group should be optional?

Reply:
You're right, I got myself confused; too many question marks, both code-wise and conceptionally...

So `(?:charset=utf-8;)?` would be correct, yes? However, I'll hold off on fixing that until we've decided whether we really wanna stick with RegEx here.

Reply:
> So `(?:charset=utf-8;)?` would be correct, yes?
Yes it would.

Node’s `url` module cannot parse data URIs, so we’d have to find and use an npm package instead.

Reply:
In that case, perhaps the following pattern would suffice:

```javascript
let regex = /sourceMappingURL=data:application\/json;(?:.*?;)base64,(\S+)/;
```

Reply:
…or we could apply two patterns in sequence:

```js
let findUrl = /sourceMappingURL=data:(\S+)/;
let findMap = /;base64,(\S+)$/;

module.exports = (fileName, attributes) => {
	let result = findUrl.exec(readFileSync(fileName));
	result = findData.exec(result[1]);
	if(!result) {
		throw new Error("No Source Map found");
	}
	let sourceMapBuffer = Buffer.from(result[1], "base64");
```